### PR TITLE
changes to part I text

### PR DESCRIPTION
**Pull request by JFRudzinski**
@ndaelman-hu - ok Nathan, check out my changes and provide comments or let me know when to merge

### CODE REVIEW

File: docs/part1.md [hunk 1]
```diff
@@ -187,19 +185,15 @@ Finally, remove all data with additional elements by selecting the **"only compo
...
 
 ??? success
     Glossing over the remaining 6 results, 3 by Andrew Rosen and 3 by Sebastian Baugmart, we retrieve **2 more MD simulations** and **2 more "Geometry Optimizations"**.
-    Even though some entries do not have a clear specification, we can find their classification by visiting the entry's overview page. <!-- @Joseph is this a bug? -->
-    While geometry optimization was not part of match our initial objective, these entries could still provide us with valuable **insights into the forcefield** setup.
-    They also inform us of the **overarching workflow** used by both authors, .e., relaxing the system before the production simulation.
-    Annotating such workflows is covered more extensively in [part II](part2.md).
+    You may notice that some entries do not have a clear specification. These entries likely need to be reprocessed to classify them correctly, since the molecular dynamics support has been developed more recently. However, you can find their classification by visiting the entry's overview page. <!-- @Joseph is this a bug? -->
```
Comment (ndaelman-hu) [inline docs/part1.md]: you still can find out their classification

File: docs/part1.md [hunk 2]
```diff
@@ -187,19 +185,15 @@ Finally, remove all data with additional elements by selecting the **"only compo
...
-
-To see this effect in action, rewind back to where we were choosing our priorities and let us go with the latter instead.
-The fastest way here, is to retrieve **"Molecular Dynamics" > "Available Properties"** and **click the highlighted checkbox** again to deselect it.
-
+<!-- This insight comes with a small caveat: the **statistics** that we see as we are stacking up our filters **are contingent on the history**.
+If we take a different route, we will see different intermediate results and this could lead us to decide on different filters. -->
+Let's go back to when we were selecting the relevant option under the Molecular Dynamics filter (click the `x` next to the `temperature chip` or go to **"Molecular Dynamics" > "Available Properties"** and **click the highlighted checkbox** again to deselect it).
 
 ??? success
     Glossing over the remaining 6 results, 3 by Andrew Rosen and 3 by Sebastian Baugmart, we retrieve **2 more MD simulations** and **2 more "Geometry Optimizations"**.
```
Comment (ndaelman-hu) [inline docs/part1.md]: 3 by Jannik Mehlis and 3 by Sebastian Baugmart

File: docs/part1.md [hunk 3]
```diff
@@ -187,19 +185,15 @@ Finally, remove all data with additional elements by selecting the **"only compo
...
-    Even though some entries do not have a clear specification, we can find their classification by visiting the entry's overview page. <!-- @Joseph is this a bug? -->
-    While geometry optimization was not part of match our initial objective, these entries could still provide us with valuable **insights into the forcefield** setup.
-    They also inform us of the **overarching workflow** used by both authors, .e., relaxing the system before the production simulation.
-    Annotating such workflows is covered more extensively in [part II](part2.md).
+    You may notice that some entries do not have a clear specification. These entries likely need to be reprocessed to classify them correctly, since the molecular dynamics support has been developed more recently. However, you can find their classification by visiting the entry's overview page. <!-- @Joseph is this a bug? -->
+    While geometry optimization was not part of match our initial objective, these entries may inform us about the **overarching workflow** used by both authors, .e., relaxing the system before the production simulation.
```
Comment (ndaelman-hu) [inline docs/part1.md]: i.e., relaxing the system before the production simulation.

File: docs/part1.md [hunk 4]
```diff
@@ -353,13 +350,13 @@ Instead, we will keep the last element a bit more open and just specify the numb
 Type it into the search bar and select the appropriate filter name.
 We want to constrain the formula, but let us keep room for a wild card, e.g. another metal or ligand constituent.
 If it trouble us down the road, we can just tighten the filter.
-So finish your _simple LTE / GTE query_ with **"<=6"**.
+So finish the query with **"<=6"**.
```
Comment (ndaelman-hu) [inline docs/part1.md]: So, here I would leave the former statement:

There are 4 query types supported by the search bar now. These each have a name (inside the code), though we haven't been using the terminology too much.

Still, I think it is good to showcase all of them, with their names. This provides a mental structure, rather than an "everything goes"-attitude, as one would have with a Google Search Bar. Knowing the names also helps when giving instructions later on. We actually refer back at some point to equality queries.

We could, however, see to change the naming to be clearer, e.g. _single inequality query_ and later on _double inequality query_.

File: docs/part1.md [hunk 5]
```diff
@@ -353,13 +350,13 @@ Instead, we will keep the last element a bit more open and just specify the numb
...
 
 The current query will leave room for systems without any metal atom, i.e. not MOFs.
 **Recreate the previous query**, but hold off on pressing the enter key.
-Rather, you should constrain it with a **lower limit** as well.
-You can add one by placing your **cursor at the beginning** and writing a similar comparison.
-Like this, you have constructed the most complex query in NOMAD, a _sandwiched LTE / GTE_.
+Rather, you should constrain it with a **lower limit** as well. In particular, let's require at least 5 different elements.
+You can add a lower limit by placing your **cursor at the beginning** and writing a similar comparison.
+<!-- Like this, you have constructed the most complex query in NOMAD, a _sandwiched LTE / GTE_. -->
```
Comment (ndaelman-hu) [inline docs/part1.md]: Same remark here. Suggested name: _double inequality query_

File: docs/part1.md [hunk 6]
```diff
@@ -418,12 +414,12 @@ For your answer, **take a look at the side menu**.
...
-    In practice, when doing **machine learning**, you would only choose a **single functional** (HSE06 in this case).
-    The only exception would be _transfer learning_, but even then you are advised to split the functionals out over separate queries.
-    Here, we look for 2 functionals just for educational reasons.
+    <!-- In practice, when doing **machine learning**, you would only choose a **single functional** (HSE06 in this case).
+    The only exception would be _transfer learning_, but even then you are advised to split the functionals out over separate queries. -->
+    Note: In practice you would only choose a **single functional** when doing **machine learning**. Here, we look for 2 functionals just for educational reasons.
 
 ???+ info "Multiple density functionals per entry"
-    A single entry (and even calculation) may contain multiple functional names, just not `XC`!
+    A single entry (and even calculation) may contain multiple functional names, not just `XC`!
```
Comment (ndaelman-hu) [inline docs/part1.md]: No, the original statement is correct: a functional either contains 1 XC, or a X and C.
As with anything that people come up with, there are other (wilder) combinations out there, but they're fairly uncommon.
We could also just leave out the `XC` specification at the end. Either way, I doubt this audience will care much.

File: docs/part1.md [hunk 7]
```diff
@@ -433,13 +429,12 @@ These can all be found under the filter subgroup "Precision".
...
-
-This feature is quite recent.
+For a full rundown on these newer features, feel free to check out FAIRmat [Tutorial 10](https://www.fairmat-nfdi.eu/events/fairmat-tutorial-10/tutorial-10-home).
+<!-- This feature is quite recent.
 For it to have a significant impact, the NOMAD database has to run over all of 13 million entries and reprocess them.
 In other words, new features will always lag behind in old data.
 Please bear this in mind.
-Since there are too few examples at the moment, we will skip this set of filters.
-If you are interested, though, feel free to check out FAIRmat [Tutorial 10](https://www.fairmat-nfdi.eu/events/fairmat-tutorial-10/tutorial-10-home) for a full rundown.
+Since there are too few examples at the moment, we will skip this set of filters. -->
```
Comment (ndaelman-hu) [inline docs/part1.md]: Yeah, explaining about reprocessing might be off-putting to would-be users. We can leave it for when they are talking to us one-on-one

File: docs/part1.md [hunk 8]
```diff
@@ -572,12 +569,16 @@ Jumping between side panes is a bit of a hassle, so instead we will speed up our
...
 
 - the periodic table ("Elements / Formula" > "Elements")
 - "Elements / Formula" > "Number of Elements"
 - "Author / Origin / Dataset" > "Dataset Name"
-- ~~"Electronic" > "Electronic Properties"~~ is not necessary, since there are only DOS
+
+<!-- And **remove from the dashboard**:
+
+- "Electronic" > "Electronic Properties" - not necessary, since there are only DOS
+@ndaelman-hu I didn't understand this, you had it striked out so I thought you wanted to remove it, but I guess you were just saying it was not necessary to add?? -->
```
Comment (ndaelman-hu) [inline docs/part1.md:607]: Exactly, it was following the filter list from scenario 2 (since scenario 3 jumps off from there), but with the DOS left out.
I could either be more explicit and say the list of filters in scenario 2 (enumerating them) and leaving the DOS striked out, or we just remove it.

File: docs/part1.md [hunk 9]
```diff
@@ -699,9 +700,9 @@ Apparently there is _cited work (no. 147)_ that shows the effectiveness of HLE17
 As a **last reflection**, note how many of the entries and statistics match our findings in scenario 2.
 Indeed, Andrew Rosen made a big contribution to our coverage of MOFs.
 Such **contributions from the community** are what drive NOMAD.
-One moment to consider contributing yourself could be right before submitting a publication, or maybe even earlier on, when analyzing your data.
+You may consider contributing your data already during the research/analysis process, right before submitting a publication, or even later on (especially for data that you have from older publications!).
 
-Andrew first even published his data over [figshare](https://figshare.com/articles/dataset/QMOF_Database/13147324) and shortly after loaded it up (see [part III](part3.md)) to NOMAD.
+Andrew first even published his data over [figshare](https://figshare.com/articles/dataset/QMOF_Database/13147324) and shortly after uploaded it to NOMAD.
```
Comment (ndaelman-hu) [inline docs/part1.md]: over at figshare

**Review by ndaelman-hu — COMMENTED** [review]
Overall, I agree with your changes: they make for a much more fluent reading experience.
Still, a couple of changes should be revoked, you will see which.

I can see to apply those changes (just answer with a thumbs-up or "okay").
I could also apply my changes to part 1 (scenario 2, the presence query, in particular) on the same branch. Afterwards, we merge it.

File: docs/part1.md (hunk 1 above)
Comment (JFRudzinski) [reply docs/part1.md]: "you can still find out..." but yes 👍

File: docs/part1.md (hunk 2 above)
Comment (JFRudzinski) [reply docs/part1.md]: 👍

File: docs/part1.md (hunk 3 above)
Comment (JFRudzinski) [reply docs/part1.md]: 👍

File: docs/part1.md (hunk 4 above)
Comment (JFRudzinski) [reply docs/part1.md]: 👍

File: docs/part1.md (hunk 5 above)
Comment (JFRudzinski) [reply docs/part1.md]: 👍

File: docs/part1.md (hunk 6 above)
Comment (JFRudzinski) [reply docs/part1.md]: oh sorry, I just misread it. You can put it back it's fine

File: docs/part1.md (hunk 7 above)
Comment (JFRudzinski) [reply docs/part1.md]: I think a lot of the stuff I commented out could be put back in for longer term use of this tutorial. It's just for this event I was trying to simplify things.

File: docs/part1.md (hunk 8 above)
Comment (JFRudzinski) [reply docs/part1.md:607]: I think just leave it out. At least for me, some of the references back to Scenario 2 were not so obvious.

File: docs/part1.md (hunk 9 above)
Comment (JFRudzinski) [reply docs/part1.md]: 👍

**Comment by JFRudzinski** [comment]
@ndaelman-hu  Can you make the changes and then merge?

**Comment by ndaelman-hu** [comment]
Applied all of my changes + polishing/
